### ISSUE TITLE
Support HTTPS with custom TLS-Certificate 

### ISSUE THREAD

**Issue by lagset** [phantomate/Untare#48]
### Your Request

Hi there, I am currently experimenting with tandoor on my NAS at home and stumbled upon this great client app. Awesome work!

I have the following setup:
Tandoor is hosted via docker and behind a reverse proxy who handles HTTPS using a custom certificate which is in turn signed by a custom Root CA created by myself. This works well with many other services I host and does so with tandoor as well, when accessing it via browser.

My Android/iOS clients trust the custom Root CA on OS level, so I don't get any caution disclaimers when accessing my services.

However, when trying to connect to tandoor in untare via HTTPS I get the following error:

![Screenshot_20230910-184557](https://github.com/phantomate/Untare/assets/15156011/b4b9088c-6db7-468b-862b-f0ef07cfadea)

It would be great, if untare would trust my Root CA as well or at least gave me the option to manually trust the certificate provided by the server.

### The Current State

For the mean time, I will use HTTP since this works great an in the end it's only recipe data exchanged.

**Comment by phantomate** [comment]
Hi, we already have a closed issue like this (https://github.com/phantomate/Untare/issues/31). The outcome is, that I would need to make the app more insecure to make self signed certificates to work. I'm not planning on doing that so unfortunately the only way to use the app is via http or to use a trusted CA with https

**Comment by lagset** [comment]
Hey thanks for letting me know and pointing me to a relevant thread.

JFYI, maybe it's relevant for you: The error I documented in the original post, was not with a self-signed certificate but with a certificate signed by a (private) CA which in turn has been added to the trusted store of Root CAs on the android device running the untare app. (This setup works with other apps on my NAS, e.g. paperless-ngx (server) and paperless-mobile (client, flutter app))

So, I guess, for untare it only works with publicly available CAs?

Again, thanks for the response, and work you've put in this app. Currently I am using it via HTTP and it works great 👍
